src/libs/Navigation/linkingConfig.js

```diff
@@ -125,6 +125,9 @@ export default {
                         path: ROUTES.BANK_ACCOUNT_WITH_STEP_TO_OPEN,
                         exact: true,
                     },
+                    GetAssistance: {
+                        path: ROUTES.GET_ASSISTANCE,
+                    },
```

Comment:
Is the `Settings` modal stack the right spot for this? Or should the Get Assistance page have its own modal stack dedicated to it?

Reply:
Yep, as we discussed [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1641854769243700), the settings stack is the right place for this since it can only be accessed from within the workspace and that's where we want the back button to go back to.